OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationOptionSpinner.java

```diff
@@ -74,4 +75,18 @@ public void setItems(@NonNull List<RegistrationOption> options, @Nullable Regist
             select(defaultOption.toString());
         }
     }
+
+    @Override
+    public void onInitializeAccessibilityEvent(AccessibilityEvent event) {
+        /*
+        Whenever there's an AppCompatSpinner on screen, its read out by TalkBack even when its not
```

Comment:
**Whenever there's an AppCompatSpinner on screen, its read out by TalkBack even when its not in focus.**

could be replaced with 

**By default whenever AppCompatSpinner is displayed on screen its read out by TalkBack which is quite unnecessary for example when screen initialized its read out.**